release.yaml

```diff
@@ -0,0 +1,6 @@
+rancher-vsphere-cpi:
+  - 101.0.0+up1.2.2
```

Comment:
I think we only want to list newly introduced charts. `rancher-vsphere-cpi:101.0.0+up1.2.2` and `rancher-vsphere-csi:101.0.0+up2.5.1-rancher1` were released with 2.7.0.